### PR TITLE
RSDK-801: Change default logger to be pretty (ISO formatted)

### PR DESCRIPTION
**Pull request by cheukt**
cannot repro the other part, so just fixing this part first

### CODE REVIEW

File: web/server/entrypoint.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func RunServer(ctx context.Context, args []string, _ golog.Logger) (err error) {
 	if argsParsed.Debug {
 		logConfig = golog.NewDebugLoggerConfig()
 	} else {
-		logConfig = golog.NewProductionLoggerConfig()
+		logConfig = golog.NewDevelopmentLoggerConfig()
```
Comment (dannenberg) [inline web/server/entrypoint.go:60]: in addition to changing the timestamp format, this appears to set `Development` to true and change the EncodeLevel to `zapcore.CapitalColorLevelEncoder` from `zapcore.LowercaseLevelEncoder`.

are those safe/desirable changes?

Comment (dannenberg) [reply web/server/entrypoint.go:60]: the former change i do not understand, but seems likely undesirable (setting `Development: true` to production builds).

the latter i'm fairly confident we do not want. colors are typically implemented via escape codes and will not render correctly in many places.

Comment (cheukt) [reply web/server/entrypoint.go]: fair, did a less invasive change - just updates the time encoder to ISO

Comment (cheukt) [reply web/server/entrypoint.go]: logs look like this now `2022-11-23T16:32:27.338-0500    info    robot_server    rpc/server.go:739       stopped cleanly`

**Pull request merged**